tests/_utils/TestBed.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactElement } from 'react';
+import { IntlProvider } from 'react-intl';
+import { render } from "@testing-library/react";
+import { ThemeProvider } from 'react-polymorph/lib/components/ThemeProvider';
+import { SimpleSkins } from 'react-polymorph/lib/skins/simple';
+import { SimpleDefaults } from 'react-polymorph/lib/themes/simple';
+
+import { BrowserLocalStorageBridge, DiscreetModeFeatureProvider } from '../../source/renderer/app/features';
+import translations from '../../source/renderer/app/i18n/locales/en-US.json';
+import { daedalusTheme } from '../../source/renderer/app/themes/daedalus';
+import { themeOverrides } from '../../source/renderer/app/themes/overrides';
+
+const TestBed = ({ children }: { children: ReactElement }) => (
+  <ThemeProvider
+    theme={daedalusTheme}
+		skins={SimpleSkins}
```

Comment:
It looks like there is still some formatting issue here. We need to have a closer look at our lint config as it did not complain about it.. :/

Reply:
We can set `"indent": ["error", 2]` but I got 72 changed files. 😀